### ISSUE TITLE
The evac shuttle should always dock at evac, regardless of conditions

### ISSUE THREAD

**Issue by Flareguy** [space-wizards/space-station-14#22702]
## Description
The evac shuttle should always dock at where the evac docking airlocks are roundstart, regardless of if the airlocks are still there or if there are any obstructions. From what I've been told we already support shuttles being able to just delete tiles (see terminal station).

Currently, you can just unanchor the docking airlocks or place lattices in front of them and the shuttle will either dock in an incredibly obtuse location or like 10-50~ tiles out into deep space.
There is basically no counterplay to this, and if it docks in deep space it will usually effectively make 95% of the crew completely unable to get on.

**Comment by B1essedknight** [comment]
maybe, it should be good to have a warning system for the shuttle docking similar to ss13 to prevent accidental deletion like decals or maybe something similar to the warden's holo barrier.

**Comment by koteq** [comment]
Personally, I like the mechanic that the evac shuttle docking logic can be exploited to have it docked to a wrong place or even no place at all. It kinds of adds up to the total havoc and chaos that usually accompanies an evacuation, making it more fun. However, I do not like that everyone can exploit it. I'd love to see only antags being capable of interfering with the evac docking logic.

**Comment by Emisse** [comment]
i agree i just think if it docks off station it should be a lot closer or leave a trail somehow to get to it from an airlock
